README.md

```diff
@@ -7,18 +7,25 @@
 
 ## Installation
 
-The preferred way to install this extension is through [composer](http://getcomposer.org/download/).
+The preferred way to install this extension is through [composer](http://getcomposer.org/download/). yii2-froala editor depends on Froala PHP SDK
+To use and Install the editor, one must install PHP SDK also.
 
 Either run
 
 ```
 php composer.phar require --prefer-dist froala/yii2-froala-editor
+php composer.phar require --prefer-dist froala/wysiwyg-editor-php-sdk
+php composer.phar require --prefer-dist bower-asset/froala-wysiwyg-editor
+
 ```
 
 or add
 
 ```
-"froala/yii2-froala-editor": "^2.6.0"
+"froala/yii2-froala-editor": "^2.6.0",
+"froala/wysiwyg-editor-php-sdk" : "*",
+"bower-asset/froala-wysiwyg-editor": "^2.6.0"
```

Comment:
Same, there is a compose package: https://packagist.org/packages/froala/wysiwyg-editor.